src/pps/pps.proto

```diff
@@ -1027,6 +1027,22 @@ message SetProjectDefaultsResponse {
   repeated Pipeline affected_pipelines = 1;
 }
 
+message PipelinesSummaryRequest {
```

Comment:
Could you move these inline comments to the line above? The proto parser expects that location for comments/documentation and it would be nice to automatically have this info in the python code.